File: ScpProject/Models/Organization/StripeCustomerData.cs
namespace Models.Organization
{
    public class StripeCustomerData
    {
        public StripeCustomerData(string firstName, string lastName, string addressLine1, string addressLine2, string phone, string zip, string email, string city, string state, string country, string cardNumber, int expiryMonth, int expiryYear, int cVC, string coupon)
        {
            FirstName = firstName;
            LastName = lastName;
            AddressLine1 = addressLine1;
            AddressLine2 = addressLine2;
            Phone = phone;
            Zip = zip;
            Email = email;
            City = city;
            State = state;
            Country = country;
            CardNumber = cardNumber;
            ExpiryMonth = expiryMonth;
            ExpiryYear = expiryYear;
            CVC = cVC;
            Coupon = coupon;    
        }

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string Phone { get; set; }
        public string Zip { get; set; }
        public string Email { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
        public string CardNumber { get; set; }
        public int ExpiryMonth { get; set; }
        public int ExpiryYear { get; set; }
        public int CVC { get; set; }
        public string Coupon { get; set; }
    }
}

File: ScpProject/StrengthConApp/src/app/Components/stripe-data/StripeCustomerData.ts
export class StripeCustomerData {
    public FirstName: string;
    public LastName: string;
    public AddressLine1: string;
    public AddressLine2: string;
    public Phone: string;
    public Zip: string;
    public Email: string;
    public City: string;
    public State: string;
    public Country: string;
    public CardNumber: string;
    public ExpiryMonth: number;
    public ExpiryYear: number;
    public CVC: number;
    public Coupon: string;
    public Adddress1:string;
    public Adddress2:string;
    
    constructor(FirstName,
        LastName,
        AddressLine1,
        AddressLine2,
        Phone,
        Zip,
        Email,
        City,
        State,
        Country,
        CardNumber,
        ExpiryMonth,
        CVC, ExpiryYear, Coupon) {
        this.FirstName = FirstName;
        this.LastName = LastName;
        this.AddressLine1 = AddressLine1;
        this.AddressLine2 = AddressLine2;
        this.Phone = Phone;
        this.Zip = Zip;
        this.Email = Email;
        this.City = City;
        this.State = State;
        this.Country = Country;
        this.CardNumber = CardNumber;
        this.ExpiryMonth = ExpiryMonth;
        this.ExpiryYear = ExpiryYear;
        this.CVC = CVC;
        this.Coupon = Coupon;
    }
}
